src/test/resources/conf/parallel.conf.json

```diff
@@ -0,0 +1,27 @@
+{
+  "server": "hub.browserstack.com",
+  "user": "BROWSERSTACK_USERNAME",
+  "key": "BROWSERSTACK_ACCESS_KEY",
+
+  "capabilities": {
+    "build": "cucumber-java-browserstack",
+    "name": "parallel_test",
```

Comment:
`name` is not being used, as the session name is getting set from the name of scenario here: 
 https://github.com/abhi2810/cucumber-java-browserstack/blob/4c30f991c3daf5f41e03f697c1c8a464feeefedf/src/test/java/com/browserstack/stepdefs/SearchSteps.java#L36
 If this is the desired behaviour, we can remove `name` from here (and other conf files)